client/src/app/core/repositories/relations.ts

```diff
@@ -338,117 +344,136 @@ export const RELATIONS: Relation[] = [
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionCommentSection,
         OField: 'motion_comment_sections',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionComment,
         OField: 'motion_comments',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionCategory,
         OField: 'motion_categories',
         OIdField: 'motion_category_ids',
         MField: 'meeting',
-        order: 'weight'
+        order: 'weight',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionBlock,
         OField: 'motion_blocks',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionSubmitter,
         OField: 'motion_submitters',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionChangeRecommendation,
         OField: 'motion_change_recommendations',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionWorkflow,
         OField: 'motion_workflows',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionState,
         OField: 'motion_states',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewMotionStatuteParagraph,
         OField: 'motion_statute_paragraphs',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewPoll,
         OField: 'polls',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewOption,
         OField: 'options',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewVote,
         OField: 'votes',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
         OViewModel: ViewMeeting,
         MViewModel: ViewAssignment,
         OField: 'assignments',
-        MField: 'meeting'
+        MField: 'meeting',
+        MisFullList: true
     }),
     ...makeM2O({
```

Comment:
You only use the M-side of M2O and M2M. Please adjust the definition of M2= to just accept `isFullList` (without M) since the O-side is never a list. Also O2O is not used -> remove it.